app/jobs/story_update_job.rb

```diff
@@ -17,6 +17,9 @@ def perform(*args)
   end
 
   def receive_story_update(data)
+    # don't allow invalid episodes to update or publish
+    return if ['update', 'publish'].include?(action) && data[:status] == 'invalid'
```

Comment:
Hmm... what status does an `uploaded` / `failed` / `notfound` audio_file end up setting on the story?  We should catch those cases as well.